<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q_SP9500X_FPGA_Widget</class>
 <widget class="QWidget" name="Q_SP9500X_FPGA_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox{ border:1px solid black; margin-top:0.5ex; }
QGroupBox::title { subcontrol-origin:margin; subcontrol-position:top left; top:-1px; left:10px; margin-left: 0px; padding:0 1px; }</string>
       </property>
       <property name="title">
        <string>RFU_VU9P</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Offset(Hex)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOffsetRFU_V9"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonWriteRFU_V9">
            <property name="text">
             <string>Write</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Data(Hex)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditDataRFU_V9"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonReadRFU_V9">
            <property name="text">
             <string>Read</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStatusRFU_V9">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">QGroupBox{ border:1px solid black; margin-top:0.5ex; }
QGroupBox::title { subcontrol-origin:margin; subcontrol-position:top left; top:-1px; left:10px; margin-left: 0px; padding:0 1px; }</string>
       </property>
       <property name="title">
        <string>BBU_VU9P</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Offset(Hex)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOffsetBBU_V9"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonWriteBBU_V9">
            <property name="text">
             <string>Write</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Data(Hex)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditDataBBU_V9"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonReadBBU_V9">
            <property name="text">
             <string>Read</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStatusBBU_V9">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="styleSheet">
        <string notr="true">QGroupBox{ border:1px solid black; margin-top:0.5ex; }
QGroupBox::title { subcontrol-origin:margin; subcontrol-position:top left; top:-1px; left:10px; margin-left: 0px; padding:0 1px; }</string>
       </property>
       <property name="title">
        <string>Spartan 6</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Offset(Hex)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOffsetS6"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonWriteS6">
            <property name="text">
             <string>Write</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Data(Hex)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditDataS6"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonReadS6">
            <property name="text">
             <string>Read</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStatusS6">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSpartan6_2">
       <property name="styleSheet">
        <string notr="true">QGroupBox{ border:1px solid black; margin-top:0.5ex; }
QGroupBox::title { subcontrol-origin:margin; subcontrol-position:top left; top:-1px; left:10px; margin-left: 0px; padding:0 1px; }</string>
       </property>
       <property name="title">
        <string>AD998X</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutRx6_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelS6_Addr_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Addr(Hex)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAD998X_Addr">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>5</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonWriteAD998X">
            <property name="text">
             <string>Write</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutRx6_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelS6_Data_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Data(Hex)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAD998X_Data">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>5</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonReadAD998X">
            <property name="text">
             <string>Read</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>24</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="styleSheet">
      <string notr="true">QGroupBox{ border:1px solid black; margin-top:0.5ex; }
QGroupBox::title { subcontrol-origin:margin; subcontrol-position:top left; top:-1px; left:10px; margin-left: 0px; padding:0 1px; }</string>
     </property>
     <property name="title">
      <string>Program</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_17">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QTextEdit" name="lineEditPathRFU_V9"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <widget class="QPushButton" name="pushButtonSetPathRFU_V9">
            <property name="text">
             <string>RFU_VU9P_bit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonProgramRFU_V9">
            <property name="text">
             <string>Program</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QTextEdit" name="lineEditPathBBU_V9"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QPushButton" name="pushButtonSetPathBBU_V9">
            <property name="text">
             <string>BBU_VU9P_bit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonProgramBBU_V9">
            <property name="text">
             <string>Program</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
